.stylelintrc.yml

```diff
@@ -13,6 +13,8 @@ rules:
   unit-disallowed-list:
     - em
     - rem
+  at-rule-no-unknown: null
+  scss/at-rule-no-unknown: true
```

Comment:
As we discussed in slack. Try to get rid of that and use extends scss. Let me know if it does not work